joomla/joomla-cms
# [4.2] Tinymce editor.css

Ensures that a non-minified editor.css will be used if present even if debug is OFF

To test
1. Open an article for editing
2. Use your browser tools eg chrome inspector to verify that media\templates\site\cassiopeia\css\editor.min.css is loaded
3. Delete that file and check again. This time you should get the non-minified version of that file

Before this PR in step 3 you get the minified version from the system eg media\system\css\editor.min.css

After this PR in step 3 you correctly get media\templates\site\cassiopeia\css\editor.css

Pull Request for Issue (sorry I can't find it) 

No documentation change required as this is fixing a bug to make the code work the way it is documented.

### Link to documentations
Please select:
